packages/api-db-mongodb/src/migration.ts

```diff
@@ -836,23 +830,23 @@ export const Migrations: Migration[] = [
 
       for (const article of migrationArticles) {
         if (article.draft) {
-          article.draft.blocks.forEach((block: ArticleBlock) => {
+          article.draft.blocks.forEach((block: BlockWithoutJSON) => {
```

Comment:
Prisma does not support Union types and to keep the blocks typed and compatible to the current typings I had to add `Prisma.JsonValue` to the Blocks. As this does not contain a `type` I made a type that removes that value.

This is currently a workaround for improved typings so that we do not have to type it as any when converting.